src/lib/viewers/image/Image.scss

```diff
@@ -48,3 +48,13 @@
     display: block;
     width: 100%;
 }
+
+.bp-content {
+    &.bp-annotations-ftux-image-cursor-seen {
+        .bp-image {
```

Comment:
Is the `.bp-image` a needed specificity?

Reply:
Yes it is required.

Reply:
What about if we did `&.bp-annotations--create-region.bp-annotations-ftux-image-cursor-seen`?